utilities/transactions/write_unprepared_transaction_test.cc

```diff
@@ -272,53 +346,81 @@ TEST_P(WriteUnpreparedTransactionTest, RecoveryTest) {
   }
 }
 
-TEST_P(WriteUnpreparedTransactionTest, UnpreparedBatch) {
+// Test whether logs containing unprepared/prepared batches are kept even
+// after memtable finishes flushing, and whether they are removed when
+// transaction commits/aborts.
+//
+// Similar to TransactionTest/TwoPhaseLogRollingTest.
```

Comment:
If it is very similar can you augment TransactionTest/TwoPhaseLogRollingTest instead?

Reply:
I added it here just so that we can get some test validation before I fix the transaction_test tests to exercise the unprepared code paths more.

Reply:
I'll add a todo to merge with those tests.